### ISSUE TITLE
Completion toolbar not accessible

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#172165]
Testing #172029

Ctrl+k, Ctrl+I should move focus into the completion toolbar. Or there should be some command that does this.
Then the usual toolbar navigation should work (left/right arrow, space to trigger action).

**Comment by hediet** [comment]
When the toolbar always shows, there is currently no way to focus it without using the mouse (but this is also not the default).

When it only shows in the hover, Ctrl+K, Ctrl+I should work (if it doesn't work that's a bug of the hover).
